pyartcd/pyartcd/pipelines/build_rhcos.py

```diff
@@ -58,184 +52,72 @@ def __init__(self, runtime: Runtime, new_build: bool, ignore_running: bool, vers
     def run(self):
         self.request_session.headers.update({"Authorization": f"Bearer {self.retrieve_auth_token()}"})
         current = self.query_existing_builds()
-        result = {}
-        if current and not self.ignore_running:
-            result["action"] = "skip"
-            result["builds"] = [
-                dict(url=b["url"], description=b["description"], result=None)
-                for b in current
-            ]
+        if self.dry_run:
+            print('DRY RUN - Exiting', file=sys.stderr)
         else:
-            # [lmeyer] there is a window of at least seconds after some other build is created,
-            # when the above check cannot see it, and so this may start a new one. interestingly,
-            # Jenkins seems to silently ignore identical builds in close succession (probably
-            # double-click protection - if the parameters differ both are started), and should this
-            # happen we will see a build start, assume we started it, and watch it to completion.
-            # this seems unlikely to cause any problems other than mild confusion.
-            self.start_build()
-            if self.dry_run:
-                print('DRY RUN - Exiting', file=sys.stderr)
-            result["action"] = "build"
-            result["builds"] = self.wait_for_builds()
```

Comment:
we used to start_build then wait for builds, now start_build should be able to get build id